clients/src/main/java/org/apache/kafka/common/security/ssl/SSLFactory.java

```diff
@@ -60,12 +60,13 @@ public void configure(Map<String, ?> configs) throws KafkaException {
 
         if (configs.get(SSLConfigs.SSL_CIPHER_SUITES_CONFIG) != null) {
             List<String> cipherSuitesList = (List<String>) configs.get(SSLConfigs.SSL_CIPHER_SUITES_CONFIG);
-            this.cipherSuites = (String[]) cipherSuitesList.toArray(new String[cipherSuitesList.size()]);
+            if (!cipherSuitesList.isEmpty())
+                this.cipherSuites = cipherSuitesList.toArray(new String[cipherSuitesList.size()]);
```

Comment:
Yes - I agree. I don't think overriding the JVM defaults is a good idea unless there is a specific, documented intent.